src/pkg/plug/scanner/dao/scanner/model.go

```diff
@@ -0,0 +1,120 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scanner
+
+import (
+	"encoding/json"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// Registration represents a named configuration for invoking a scanner via its adapter.
+// UUID will be used to track the scanner.Endpoint as unique ID
+type Registration struct {
+	// Basic information
+	// int64 ID is kept for being aligned with previous DB schema
+	ID          int64  `orm:"pk;auto;column(id)" json:"-"`
+	UUID        string `orm:"unique;column(uid)" json:"uid"`
+	Name        string `orm:"unique;column(name);size(128)" json:"name"`
+	Description string `orm:"column(description);null;size(1024)" json:"description"`
+	URL         string `orm:"column(url);unique;size(512)" json:"url"`
+	Disabled    bool   `orm:"column(disabled);default(true)" json:"disabled"`
+	IsDefault   bool   `orm:"column(is_default);default(false)" json:"isDefault"`
```

Comment:
maybe `json:"is_default"` for other apis all use the snake case in the response

Reply:
ok